### DISCUSSION TITLE
[Feature] Add the actual disk size for users.

### DISCUSSION THREAD

**Discussion by Plydow** [immich-app/immich#14484] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, the administration page on the space taken by each user shows the sum of the original file sizes. With Thumbnails and Previews, this can quickly increase. Especially with videos.

For example, on my personal instance:
- Total size on Immich: 27GiB
- Total size on disk (only upload folder mounted on Immich container): 253GB

It would be easier to diagnose real disk usage between our different users to have an indicator in the administration interface.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#3186
